ironfish-rust/src/public_account/transfer.rs

```diff
@@ -0,0 +1,53 @@
+use std::io;
+
+use crate::{assets::asset_identifier::AssetIdentifier, errors::IronfishError, PublicAddress};
+
+#[derive(Clone, Copy)]
+pub struct PublicMemo(pub [u8; 256]);
+
+#[derive(Clone, Copy)]
+pub struct Transfer {
+    pub(crate) asset_id: AssetIdentifier,
+    pub(crate) amount: u64,
```

Comment:
Wondering if this should be signed (`i64`)? How do we distinguish transfers in and out? Is there going to be another struct?